primitive-types/Cargo.toml

```
@@ -15,7 +15,8 @@ impl-serde = { version = "0.3.1", path = "impls/serde", default-features = false
 impl-codec = { version = "0.6.0", path = "impls/codec", default-features = false, optional = true }
 impl-num-traits = { version = "0.1.0", path = "impls/num-traits", default-features = false, optional = true }
 impl-rlp = { version = "0.3", path = "impls/rlp", default-features = false, optional = true }
-scale-info-crate = { package = "scale-info", version = ">=0.9, <2", features = ["derive"], default-features = false, optional = true }
+# See https://github.com/paritytech/parity-common/pull/556#issuecomment-872127496 why this is here:
+scale-info-crate = { package = "scale-info", version = "2", features = ["derive"], default-features = false, optional = true }
```

Ok, pull this YOLO trick once again
```suggestion
scale-info-crate = { package = "scale-info", version = ">=0.9, <4", features = ["derive"], default-features = false, optional = true }
```

---

`<3` you meant? Also, isn't it better to exclude 0.9 and 1.0.0 now?

---

With this we'd only need a minor release of `primitive-types` yes?

---

Yes, I meant <3 and yes, this would be a patch release. And we don't need to release anything else. Only update substrate/polkadot/cumulus

---

Okay, I get it. Although MSRV and edition 2021 is already in the Cargo.toml which technically makes it a major release, right?

---

Could you bump the version in this PR and update the change as well please?

---

Ah, wait, scale-info also used in ethereum-types. I'm afraid, there's no getting away with breaking release then (sorry).